src/system/RoutesModule/Entity/Repository/Base/AbstractRouteRepository.php

```diff
@@ -743,11 +742,10 @@ public function retrieveCollectionResult(Query $query, $orderBy = '', $isPaginat
      * @param boolean $useJoins Whether to include joining related objects (optional) (default=true)
      *
      * @return QueryBuilder Created query builder instance
-     * @TODO fix usage of joins; please remove the first line and test
      */
     protected function getCountQuery($where = '', $useJoins = true)
     {
-        $useJoins = false;
+        $useJoins = false; // joins usage needs to be fixed; please remove the first line and test
```

Comment:
This isn't really relevant for routes, refs https://github.com/Guite/MostGenerator/issues/977 - ignore ;-)